### PR TITLE
Add crepe, gin-config, ddsp

### PR DESCRIPTION
**Pull request by oblute**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@rluria14 @ndmaxar agree to maintain with me?

**Comment by oblute** [comment]
@conda-forge/staged-recipes this is ready for review

File: recipes/crepe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - h5py <3.0.0,>=2.7.0
+    - hmmlearn <0.3.0,>=0.2.0
+    - imageio >=2.3.0
+    - matplotlib >=2.1.0
```
Comment (xhochy) [inline recipes/crepe/meta.yaml:28]: Only depend on `matplotlib` if you really need the qt backend.

```suggestion
    - matplotlib-base >=2.1.0
```

File: recipes/gin-config/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - six >=1.10.0
+    - tensorflow
```
Comment (xhochy) [inline recipes/gin-config/meta.yaml:24]: This is an optional dependency. 
```suggestion
```

File: recipes/gin-config/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - gin
+    - gin.testdata
+    - gin.tf
```
Comment (xhochy) [inline recipes/gin-config/meta.yaml:30]: You should only check for `gin`, the others are only optional.
```suggestion
```

Comment (oblute) [reply recipes/gin-config/meta.yaml:30]: @xhochy wouldn't i want to include `tensorflow` in the build in case there is required use for `gin.tf`?

Comment (xhochy) [reply recipes/gin-config/meta.yaml]: No, python dependencies aren't necessary in the build and add them to run makes them non-optional; and especially tensorflow is quite a heavy-weight dependency which we should keep optional if possible.

Comment (xhochy) [reply recipes/gin-config/meta.yaml]: If there are only certain versions of `tensorflow` with which `gin` works together, you could add a `run_constrained` for that.-

Comment (oblute) [reply recipes/gin-config/meta.yaml]: @xhochy i went ahead and added a run constrained. should be good to go now. thanks!

**Pull request merged**